ynput/OpenPype
# ImageIO: Adding ImageIO activation toggle to all hosts

## Changelog Description
Colorspace management can now be enabled at the project level, although it is disabled by default. Once enabled, all hosts will use the OCIO config file defined in the settings. If settings are disabled, the system switches to DCC's native color space management, and we do not store colorspace information at the representative level.

## Additional info
Each host that publishes pixel data can now enable Color Management. This allows the publishing and loading process to run through file rules identification. 

## Hosts needs to be tested
- [ ] After Effects
- [ ] 3D Max
- [ ] Substance Painter

## Testing notes:
1. Open Studio Project Settings
2. Enable `project_settings/global/imageio/activate_global_color_management`
3. Go to any host and see there following  toggles each ImageIO category - except Global:
     - **Enable Color Management** - this overrides the globally enabled colormanagement.
4. each of `ocio_config` subcategory is holding now new toggle  **Override global OCIO config**
5. each of `file_rules` subcategory is holding now new toggle  **Override global file rules**

closes https://github.com/ynput/OpenPype/pull/4700

# Documentation
https://github.com/ynput/ayon-documentation/pull/42
